xs/src/libslic3r/GCode.cpp

```diff
@@ -391,23 +391,23 @@ GCode::extrude(ExtrusionLoop loop, std::string description, double speed)
         // Find the closest point, avoid overhangs.
         loop.split_at(last_pos, true);
     }
-    
+
     // clip the path to avoid the extruder to get exactly on the first point of the loop;
     // if polyline was shorter than the clipping distance we'd get a null polyline, so
     // we discard it in that case
     double clip_length = this->enable_loop_clipping
         ? scale_(EXTRUDER_CONFIG(nozzle_diameter)) * LOOP_CLIPPING_LENGTH_OVER_NOZZLE_DIAMETER
         : 0;
-    
+
     // get paths
     ExtrusionPaths paths;
     loop.clip_end(clip_length, &paths);
     if (paths.empty()) return "";
-    
+
     // apply the small perimeter speed
     if (paths.front().is_perimeter()
         && !loop.has(erOverhangPerimeter)
-        && loop.length() <= SMALL_PERIMETER_LENGTH
```

Comment:
I think you forgot to remove this constant's declaration?

https://github.com/slic3r/Slic3r/blob/fb2fefd4daa5600b7b46b428ba4278f3dc5a6e11/xs/src/libslic3r/libslic3r.h#L86